core/services/workflow_service/schemas/project.py

```diff
@@ -0,0 +1,51 @@
+from uuid import UUID
+
+from pydantic import BaseModel
+from datetime import datetime
+from typing import List
+
+
+# TODO:Still missing any field validation (empty name, uuid is not UUID..)
```

Comment:
nit: `uuid is not UUID` is actually beeing checked, can be removed from the TODO